frontend/packages/console-shared/src/components/spotlight/Spotlight.scss

```diff
@@ -0,0 +1,91 @@
+@keyframes highlight {
+  0% {
+    opacity: 1;
+    border: var(--pf-global--BorderWidth--xl) solid var(--pf-global--palette--blue-200);
```

Comment:
Why didn't we use box shadow here as shown in https://codesandbox.io/s/highlight-9oizs

Reply:
the UX for animation was updated
please find the updated UX for animation [here](https://xd.adobe.com/view/dd20d755-8def-4536-af2a-86ba83a8080c-8873/)
Slack Conversation links: 
https://coreos.slack.com/archives/C014NQG3H5L/p1604609753173100
https://coreos.slack.com/archives/C014NQG3H5L/p1604610208176400